BuildLoader.py

```diff
@@ -1378,26 +1379,31 @@ def main():
     prep_env ()
     board_cfgs   = []
     board_names  = []
+    module_names = []
 
     # Find all boards
     search_dir = os.environ['SBL_SOURCE']
     if 'PLT_SOURCE' in os.environ:
         search_dir = os.path.abspath(os.path.join(search_dir, os.path.pardir))
     board_pkgs = os.listdir (search_dir)
+    board_cfgs = []

```

Comment:
This is not required since it is done in line 1380.

Reply:
Thanks. Removed the unnecessary line.